### PR TITLE
#2 Replace elasticsearch with opensearech and make test pass

### PR DESCRIPTION
**Pull request by chirag4semandex**
- Replaced elastic search with open search dependencies
- Refactor the code to take the connection as an argument
- make sure the test passes
- Prepared the setup.py to make the release

### CODE REVIEW

File: mordecai3/geoparse.py [hunk 1]
```diff
@@ -29,8 +29,12 @@
 
 spacy_doc_setup()
 
+spacy_model_name = os.environ.get("SPACY_MODEL_NAME", "en_core_web_trf")
+mordecai_model_pt = "assets/mordecai_2024-06-04.pt"
```
Comment (djmakwana) [inline mordecai3/geoparse.py:33]: do you know what is this file for?

**Review by djmakwana — APPROVED** [review]

File: mordecai3/geoparse.py (hunk 1 above)
Comment (chirag4semandex) [reply mordecai3/geoparse.py:33]: No idea, looks like some kinda model they have .. very small but it looks like a model

File: mordecai3/geonames_loader.py [hunk 2]
```diff
@@ -282,24 +283,25 @@ def load_geocodes(self):
...
-#
-#     csv.field_size_limit()
-#
-#     loader = GeoNamesLoader(index_name=index, os_client=client, data_dir=geo_dir)
-#     loader.load_geocodes()
-#
-#     e = (time.time() - t) / 60
-#     logger.info("Total tile loading in minutes: ", e)
+
+if __name__ == "__main__":
```
Comment (ianrandman) [inline mordecai3/geonames_loader.py:287]: When is this used?

Comment (chirag4semandex) [reply mordecai3/geonames_loader.py:287]: Original code as such but its only used for testing I believe

**Review by ianrandman — APPROVED** [review]

**Pull request merged**